src/style.css

```diff
@@ -1 +1,102 @@
-/* Styles go here */
+
+* {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  color: #000;
+  padding: 0;
+  margin: 0;
+  
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.main-header {
+  display: none;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: absolute;
+  width: 1200px;
+  left: 0;
+  top: 0;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  left: 50px;
+  top: 10px;
+}
+
+.nav__list {
+  padding-left: 0;
+  display: flex;
+  list-style: none;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.nav-item {
+  margin-right: 20px;
+  list-style: none;
+  position: relative;
+  text-transform: uppercase;
+  text-decoration: none;
+  align-items: flex-end;
+  padding-bottom: 19px;
+  padding-top: 19px;
+
+}
+
+.nav-item:last-child {
+  margin-right: 0;
+}
+
+.nav_tag {
+  /* position: absolute; */
+  text-decoration: none;
+  padding-bottom: 19px;
+  padding-top: 19px;
+  font-size: 12px;
+  height: 15px;
+  top: 22.5px;
+
+}
+
+.nav_tag:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active:after {
+  position: absolute;
+  content: "";
+  height: 4px;
+  bottom: -4px;
+  background: #00acdc;
+  top: 56px;
```

Comment:
you wont require to have top positioning if you have proper bottom position here